package.json

```diff
@@ -117,7 +117,7 @@
     ]
   },
   "lint-staged": {
-    "**/*.{ts?(x),?(m)js} !(**/stencil-generated/*.ts)": [
+    "**/*.{ts?(x),?(m)js, !(*.ts|stencil-generated/&.ts)}": [
```

Comment:
What does `&` stand for? 
Why are `*.ts` files being excluded?

Reply:
Sorry, I pushed and created the pr too fast. This doesnt work. I wanted to revert the pr to 'draft' mode but I cant since its already approved. This version is not working. I will re-request reviews when it works. 

The reason for these files being excluded is that they are ignored in the eslint config but eslint adds a 'warning' whenever you ignore a file. It is really dumb. see this issue for more context https://github.com/eslint/eslint/issues/15010